pkg/cmd/server/apis/config/v1/conversions.go

```diff
@@ -200,7 +200,7 @@ func SetDefaults_DockerConfig(obj *DockerConfig) {
 		obj.ExecHandlerName = DockerExecHandlerNative
 	}
 	if len(obj.DockerShimSocket) == 0 {
-		obj.DockerShimSocket = "/var/run/dockershim.sock"
+		obj.DockerShimSocket = "unix:///var/run/dockershim.sock"
```

Comment:
Does the unprefixed version not work or work differently? Do we need to add fixup for already existing config values?